Automattic/woocommerce-payments
# Update Connect Page

Fixes #1436

#### Changes proposed in this Pull Request

Updates the connect page with a new design.

![image](https://user-images.githubusercontent.com/9312929/115040188-c001e500-9f03-11eb-893f-110a2159a746.png)

![image](https://user-images.githubusercontent.com/9312929/115040238-cdb76a80-9f03-11eb-95b1-7a9ea691ab2f.png)


#### Testing instructions

With a new installation of WC Pay,

- Navigate to Payments
- See that it renders like the design in issue #1436
- See that it responds to mobile width
- Click the `Finish Setup` button
- Observe the connection screen (Note: it might error locally with a message "There was a problem connecting this site to WordPress.com")
- Observe that tracks events are created for:
  - Viewing the page
  - Clicking the `Learn More` link
  - Clicking the `Finish Setup` button

Note: tracks events can be observed from the Network Developer Tools tab and filtering `t.gif`

<img width="1177" alt="Screen Shot 2021-04-16 at 10 30 19 pm" src="https://user-images.githubusercontent.com/9312929/115039899-75806880-9f03-11eb-9122-86e550053383.png">


*

-------------------
